web-animations-1/Overview.bs

```diff
@@ -628,6 +628,21 @@ a document's load event.
 A <a>timeline</a> is considered to be <dfn lt="inactive timeline">inactive</dfn>
 when its <a>time value</a> is <a>unresolved</a>.
 
+A [=timeline=] may be in one of four possible <dfn>phase</dfn>s:
```

Comment:
Would it be better to scope phase to something like "timeline phase"?

e.g. `<dfn lt="timeline phase">phase</dfn>`.

Then you can link to it as `[=timeline phase=]` or `[=timeline phase|phase=]` for when repeating "timeline" is unnecessary. (And if that becomes too onerous we can add `local-lt="phase"`.)